### PR TITLE
Adding the Mac-Morpho dataset

### PR DESCRIPTION
**Pull request by jonatasgrosman**
Adding the Mac-Morpho dataset, a Portuguese language dataset for Part-of-speech tagging tasks

### CODE REVIEW

File: datasets/mac_morpho/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- original
+task_categories:
+- structure-prediction
+task_ids:
+- part-of-speech-tagging
```
Comment (lhoestq) [inline datasets/mac_morpho/README.md:19]:
```suggestion
- structure-prediction-other-part-of-speech-tagging
```
we've not added it yet to the taxonomy. 
I'll update this readme once it's done

File: datasets/mac_morpho/mac_morpho.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    VERSION = datasets.Version("3.0.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "pos": datasets.Sequence(
```
Comment (lhoestq) [inline datasets/mac_morpho/mac_morpho.py:81]:
```suggestion
                    "pos_tags": datasets.Sequence(
```
can you rename it pos_tags for consistency with other POS tagging datasets ?

You'll need to regenerate the dataset_infos.json file after this change

File: datasets/mac_morpho/README.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+### Languages
+
+Portuguese
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/mac_morpho/README.md:89]: can you provide one example from the dataset here please ?

You can get one by using `load_dataset` and taking the first sample `dataset["train"][0]` for example

**Review by lhoestq — COMMENTED** [review]
Thank you for adding this one !

I left a few comments

File: datasets/mac_morpho/README.md (hunk 1 above)
Comment (jonatasgrosman) [reply datasets/mac_morpho/README.md]: Done!

File: datasets/mac_morpho/mac_morpho.py (hunk 2 above)
Comment (jonatasgrosman) [reply datasets/mac_morpho/mac_morpho.py]: Done!

File: datasets/mac_morpho/README.md (hunk 3 above)
Comment (jonatasgrosman) [reply datasets/mac_morpho/README.md]: Done!

Comment (jonatasgrosman) [reply datasets/mac_morpho/README.md]: Hi @lhoestq, I have a doubt. Why do I need to provide an instance example of this dataset but not for the others I've added? Is there some particularity about this kind of dataset?

Comment (lhoestq) [reply datasets/mac_morpho/README.md]: We should have an example in as many dataset cards as possible. We've recently noticed that it was missing in too many dataset cards so we're now asking them explicitly. Feel free to add them on the other datasets you've added though.

**Review by lhoestq — APPROVED** [review]
Looks all good thanks !

**Pull request merged**